integration_tests/run.sh

```diff
@@ -36,47 +36,51 @@ cd "./integration_tests"
 
 # build and zip recorder extension
 cd recorder-extension
-    GOOS=linux GOARCH=amd64 go build -o extensions/recorder-extension main.go
-    zip -rq ext.zip extensions/* -x ".*" -x "__MACOSX" -x "extensions/.*"
+GOOS=linux GOARCH=amd64 go build -o extensions/recorder-extension main.go
+zip -rq ext.zip extensions/* -x ".*" -x "__MACOSX" -x "extensions/.*"
 cd ..
 
+go_test_dirs=("with-ddlambda" "without-ddlambda" "timeout" "log" "trace")
+
 # build Go Lambda function
 cd src
-env GOOS=linux go build -ldflags="-s -w" -o ../bootstrap traceGo.go
+for go_dir in "${go_test_dirs[@]}"; do
+    env GOOS=linux go build -ldflags="-s -w" -o bin/"$go_dir" go-tests/"$go_dir"/main.go
+done
 cd ..
 
 if [ -z "$NODE_LAYER_VERSION" ]; then
-   echo "NODE_LAYER_VERSION not found, using the default"
-   export NODE_LAYER_VERSION=$DEFAULT_NODE_LAYER_VERSION
+    echo "NODE_LAYER_VERSION not found, using the default"
+    export NODE_LAYER_VERSION=$DEFAULT_NODE_LAYER_VERSION
 fi
 
 if [ -z "$PYTHON_LAYER_VERSION" ]; then
-   echo "PYTHON_LAYER_VERSION not found, using the default"
-   export PYTHON_LAYER_VERSION=$DEFAULT_PYTHON_LAYER_VERSION
+    echo "PYTHON_LAYER_VERSION not found, using the default"
+    export PYTHON_LAYER_VERSION=$DEFAULT_PYTHON_LAYER_VERSION
 fi
 
 echo "NODE_LAYER_VERSION set to: $NODE_LAYER_VERSION"
 echo "PYTHON_LAYER_VERSION set to: $PYTHON_LAYER_VERSION"
 
 # random 8-character ID to avoid collisions with other runs
-stage=$(xxd -l 4 -c 4 -p < /dev/random)
+stage=$(xxd -l 4 -c 4 -p </dev/random)
 
 function remove_stack() {
     echo "Removing stack for stage : ${stage}"
-    serverless remove --stage ${stage} 
+    serverless remove --stage ${stage}
 }
 
 # always remove the stacks before exiting, no matter what
-trap remove_stack EXIT
+# trap remove_stack EXIT
```

Comment:
I think we still want this line uncommented

Reply:
Ahh yeah forgot I did that. Thanks!